Makefile.am

```diff
@@ -895,7 +897,9 @@ if ENABLE_STATIC
 test_apps += \
 	unittests/unittest_poller \
 	unittests/unittest_ypipe \
-	unittests/unittest_mtrie
+	unittests/unittest_mtrie \
+	unittests/unittest_ip_resolver \
+	unittests/unittest_ip_resolver_dns
 
 unittests_unittest_poller_SOURCES = unittests/unittest_poller.cpp
```

Comment:
@simias for automake you also need an explicit section like these ones, just copy and paste
that's because each test is a program